app/views/checklist_tests/_checklist_measures.html.erb

```diff
@@ -0,0 +1,125 @@
+<% if @product_test.tasks[0].most_recent_execution && @product_test.tasks[0].most_recent_execution.incomplete? %>
+<script>
+  $.ajax({url: "<%= request.env['PATH_INFO'] %>", type: "GET", dataType: 'script', data: { partial: 'checklist_measures' }});
+</script>
+<% end %>
+<div id='save_options' >
+  <% unless product_test.status != 'passing' %>
+                  <%= render :partial => 'alert', locals: {
+                                                  alert_type: 'warning',
+                                                  messages: 'Saving this checklist will overide previous results.',
+                                                  confirmation: 'Yes, let me save this checklist' } %>
+<% end %>
+<% measures = product_test.measures %>
+<% if measures %>
+  <%= bootstrap_nested_form_for([product, product_test], :url => { :controller => 'checklist_tests', :action => 'update' }) do |f| %>
+    <% measures.each do |measure| %>
+      <div class = 'panel-group' id = '<%= measure.cms_id %>' disabled = true>
+        <div class = 'panel panel-default'>
+          <div class = 'panel-heading'>
+            <h1 class='panel-title'>
+              <%= "#{measure.cms_id} #{measure.name}" %>
+            </h1>
+          </div>
+          <div class = 'panel checklist-panel'>
+            <div class = 'panel-body'>
+              <table class = 'table table-sm'>
+                <thead>
+                  <tr>
+                    <th class = 'col-sm-2'>Validated in QRDA</th>
+                    <th class = 'col-sm-2'>Data Criteria</th>
```

Comment:
I'm surprised the automated tests aren't yelling at you for this, but `<th>` elements should have an additional attribute describing the scope (e.g. `<th scope="col">asdf</th>`)